### ISSUE TITLE
Unicode format for micro prefix uses wrong unicode character

### ISSUE THREAD

**Issue by MaxNoe** [astropy/astropy#9350]
It is `GREEK SMALL LETTER MU` but should be `MICRO SIGN`.

```python
>>> f'{u.micrometer:unicode}' == '\N{GREEK SMALL LETTER MU}m'
True
>>> f'{u.micrometer:unicode}' == '\N{MICRO SIGN}m'
False
```

**Comment by pllim** [comment]
If I read the emails corrected, this is going to be fixed as part of #9348 , right?

**Comment by MaxNoe** [comment]
Yes
